.github/workflows/testcontainers-suite.yml

```
@@ -13,29 +13,52 @@ on:
 
+      - name: Install gotestsum
+        run: go install gotest.tools/gotestsum@latest
+
+      - name: Create directories
+        run: |
+          mkdir -p ${{ env.REPORTS_DIR }}
+          mkdir -p ${{ env.COVERAGE_DIR }}
+
+      - name: Build binary (if required)
+        env:
+            GOOS: ${{ inputs.os }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)